--- a/_scripts/build.js	2023-01-25 11:08:56.951389599 +0100
+++ b/_scripts/build.js	2023-01-25 11:10:02.062189054 +0100
@@ -36,7 +36,7 @@
     arch = Arch.armv7l
   }
 
-  targets = Platform.LINUX.createTarget(['deb', 'zip', '7z', 'apk', 'rpm', 'AppImage', 'pacman'], arch)
+  targets = Platform.LINUX.createTarget(['dir'], arch)
 }
 
 const config = {
@@ -92,7 +92,7 @@
   linux: {
     category: 'Network',
     icon: '_icons/icon.svg',
-    target: ['deb', 'zip', '7z', 'apk', 'rpm', 'AppImage', 'pacman'],
+    target: ['dir'],
   },
   // See the following issues for more information
   // https://github.com/jordansissel/fpm/issues/1503
